tests/worker/test_workflow.py

```diff
@@ -1915,7 +1915,8 @@ def find_log(starts_with: str) -> Optional[logging.LogRecord]:
         record = find_log("Signal: signal 1")
         assert (
             record
-            and record.__dict__["workflow_info"].workflow_type == "LoggingWorkflow"
+            and record.__dict__["temporal_workflow"]["workflow_type"]
```

Comment:
Isn't this still nested? I thought the point was to make the log attributes flat in the extra field.

Reply:
I see the PR description now. Might be worth verifying that we shouldn't flatten completely while we're changing this.

Reply:
> Isn't this still nested? I thought the point was to make the log attributes flat in the extra field.

Yes it is nested. The goal wasn't to make it flat, the goal was to remove the dataclass in favor of a dict

> Might be worth verifying that we shouldn't flatten completely while we're changing this.

:+1: I am going to get user confirmation this works with DataDog before merging. Looking [here](https://docs.datadoghq.com/logs/log_collection/python/) it may not. But I don't know that we want to include dots in our keys just for DataDog.

Reply:
Confirmed this works for DataDog